src/citrine/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '0.99.3'
+__version__ = '0.99.5'
```

Comment:
I set this to .5 so that I can merge this PR in first and then follow with this one without another commit https://github.com/CitrineInformatics/citrine-python/pull/537